rust/noosphere-ns/tests/ns_test.rs

```diff
@@ -4,15 +4,16 @@ pub mod utils;
 use anyhow::Result;
 use noosphere_core::{authority::generate_ed25519_key, data::Did, view::SPHERE_LIFETIME};
 use noosphere_ns::{
-    utils::{generate_capability, generate_fact},
-    Multiaddr, NSRecord, NameSystem,
+    utils::{generate_capability, generate_fact, wait_for_peers},
+    Multiaddr, NSRecord, NameSystem, NameSystemClient,
```

Comment:
Sorry that this occurred to me late, but technically the Rust naming conventions have this to say about acronyms:

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. In snake_case, acronyms and contractions are lower-cased: is_xid_start.

https://rust-lang.github.io/api-guidelines/naming.html

So, for example, we write `IpfsClient` instead of `IPFSClient`. So, we should write `NsRecord` instead of `NSRecord`.